integration/database.go

```diff
@@ -82,7 +83,28 @@ func (cfg PostgresDBConfig) RunAsDockerContainer(containerName string) (func() e
 	if err != nil {
 		return nil, err
 	}
-	return close, err
+
+	doneC := make(chan bool)
+	// wait until the database container is running and available
+	go func() {
```

Comment:
Not worth worrying about right now, but I don't think this goroutine will stop, right? It seems like the goroutine will indefinitely check for a database connection.

We can address this when the database behavior gets moved to the toolkit.

Reply:
great call! Will fix